.drone.star

```diff
@@ -83,8 +83,8 @@ def main(ctx):
 
 def phpIntegrationTest(ctx, phpversions, coverage):
     pipelines = []
-    steps = keycloakService() + restoreOcisCache() + ocisService() + cacheRestore()
     for php in phpversions:
+        steps = keycloakService() + restoreOcisCache() + ocisService() + cacheRestore()
```

Comment:
We need to start with `steps` set to just the initial steps at the start of each loop.
If we have multiple PHP versions to generate pipelines for, the previous code would end up with more-and-more steps being added on and get errors about duplicate step names.